recipes/highs/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, collect_libs, export_conandata_patches, get
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.53.0"
+
+
+class HiGHSConan(ConanFile):
+    name = "highs"
+    description = "high performance serial and parallel solver for large scale sparse linear optimization problems"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.highs.dev/"
+    topics = ("highs", "simplex", "interior point", "solver", "linear", "programming")
```

Comment:
No need to recipe name since it doesn't improve search 

```suggestion
    topics = ("simplex", "interior point", "solver", "linear", "programming")
```

Reply:
name dropped